test/end-to-end/pages/steps.js

```diff
@@ -11,7 +11,7 @@ module.exports = function () {
     authenticateWithIdamIfAvailable: steps.IDAM.signIn,
 
     //Start application
-    startApplication: steps.startPage.startPage,
+    startApplication: steps.starteligibility.starteligibility,
```

Comment:
Should these be 'startEligibility'? -> camelCase

Reply:
yeah this one i was confused about, I first put it as startEligibility but then as things were renamed in places with starteligibility i wasn't sure...